lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -552,6 +554,8 @@ data JormungandrLayer m = JormungandrLayer
     , getInitialBlockchainParameters
         :: Hash "Genesis"
         -> ExceptT ErrGetBlockchainParams m (J.Block, BlockchainParameters)
+    , getStakeDistribution
+        :: ExceptT ErrGetStakeDistribution m J.StakeApiResponse
```

Comment:
There's no need for the wrapper here, could be `ErrNetworkUnreachable` directly to make the code more straight-forward.

Reply:
I think optimally, we would map the BFT-response `{"epoch":2521}` into a specific error, instead of throwing.

Reply:
corrected

Reply:
> we would map the BFT-response {"epoch":2521} into a specific error,

Hmmm. No. We aren't supposed to interact with a BFT node, so that's effectively an exception.